source/features/restore-file.tsx

```diff
@@ -76,30 +76,19 @@ async function restoreFile(progress: (message: string) => void, menuItem: Elemen
 	}
 }
 
-const filesRestored = new WeakSet<HTMLButtonElement>();
```

Comment:
This was needed because we kept the dropdown open, I think. Since the dropdown now closes on click, the user cannot double click. By the time they manage to open the menu and click again, it's probably done and the file is gone.